### ISSUE TITLE
Compilation error with gcc 8.2.0

### ISSUE THREAD

**Issue by guillaumechereau** [xelatihy/yocto-gl#439]
I get those errors when I try to compile the code at current version (e155ccb2)

Tried with:
gcc (Ubuntu 8.2.0-7ubuntu1) 8.2.0

```
/home/guillaume/src/yocto-gl/yocto/yocto_bvh.cpp: In lambda function:
/home/guillaume/src/yocto-gl/yocto/yocto_bvh.cpp:1487:69: error: call of overloaded ‘affine(yocto::frame3f&)’ is ambiguous
                                          inverse((affine3f)instance.frame), ray)
                                                                     ^~~~~
In file included from /home/guillaume/src/yocto-gl/yocto/yocto_bvh.h:91,
                 from /home/guillaume/src/yocto-gl/yocto/yocto_bvh.cpp:29:
/home/guillaume/src/yocto-gl/yocto/yocto_math.h:1289:24: note: candidate: ‘constexpr yocto::affine<T, 3>::affine(const yocto::mat<T, 4, 4>&) [with T = float]’
     constexpr explicit affine(const mat<T, 4, 4>& m)
                        ^~~~~~
/home/guillaume/src/yocto-gl/yocto/yocto_math.h:1271:8: note: candidate: ‘constexpr yocto::affine<float, 3>::affine(const yocto::affine<float, 3>&)’
 struct affine<T, 3> {
        ^~~~~~~~~~~~
/home/guillaume/src/yocto-gl/yocto/yocto_math.h:1271:8: note: candidate: ‘constexpr yocto::affine<float, 3>::affine(yocto::affine<float, 3>&&)’
```

**Comment by xelatihy** [comment]
I'll look into. Strange that the CI is passing.

**Comment by xelatihy** [comment]
I have added GCC8 to travis. Let us see what happens.

**Comment by guillaumechereau** [comment]
Thanks!

**Comment by xelatihy** [comment]
BTW, we will make a v1.0 release within 10 days. There may be a couple of braking changes before that to improve memory usage in very large scene (Disney Maona island).

**Comment by guillaumechereau** [comment]
Nice.

I just merged yocto-gl use into my voxel editor (goxel):

![image](https://user-images.githubusercontent.com/107679/55154360-95fbef80-518f-11e9-9d0c-960eb11b82b1.png)

So this will be available in the next release, and soon in the iOS version too!

**Comment by xelatihy** [comment]
Wow. Thanks!!! The v1.0 will soon be released. We are refactoring some code by the APIs *should* remain the same. I suggest you stick to the version you have till we lock in our changes. Once v1 is out, APIs will stay stable. 10 days to drop...

**Comment by xelatihy** [comment]
Quick question. We do not understand why this breaks the code right now. Do you need it? If not, I'd like to postpone this after 1.0.

**Comment by xelatihy** [comment]
Fixed in #451
